src/flow/src/adapter/worker.rs

```diff
@@ -109,50 +109,35 @@ impl WorkerHandle {
     /// create task, return task id
     ///
     #[allow(clippy::too_many_arguments)]
-    pub async fn create_flow(
-        &self,
-        task_id: FlowId,
-        plan: TypedPlan,
-        sink_id: GlobalId,
-        sink_sender: mpsc::UnboundedSender<DiffRow>,
-        source_ids: &[GlobalId],
-        src_recvs: Vec<broadcast::Receiver<DiffRow>>,
-        expire_when: Option<repr::Duration>,
-        create_if_not_exist: bool,
-        err_collector: ErrCollector,
-    ) -> Result<Option<FlowId>, Error> {
-        let req = Request::Create {
-            task_id,
-            plan,
-            sink_id,
-            sink_sender,
-            source_ids: source_ids.to_vec(),
-            src_recvs,
-            expire_when,
-            create_if_not_exist,
-            err_collector,
-        };
+    pub async fn create_flow(&self, create_reqs: Request) -> Result<Option<FlowId>, Error> {
```

Comment:
What about: `create_reqs: Request::Create` ?

Reply:
> What about: `create_req: Request::Create` ?

seems not possible: https://users.rust-lang.org/t/passing-enum-variant-as-argument/94422/1